_posts/0000-01-02-adonisaustin.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
Hello Im Adonis Austin Im a multi-discipline Designer. with a  modern take of design solutions